python/python/ert/ecl/faults/fault_collection.py

```diff
@@ -31,57 +31,56 @@ def dequote(s):
 
 
 class FaultCollection(object):
-    def __init__(self , grid = None , *file_list):
+    def __init__(self, grid=None, *file_list):
```

Comment:
This code now seems to mix the input argument `grid` and the member variable `self.__grid` somewhat randomly. Is the member variable `self.__grid` at all needed?